### PR TITLE
update instructions to set docker auth using org-infra

### PR DESCRIPTION
**Pull request by andrewpearce-digital**

### CODE REVIEW

**Review by mattmachell — APPROVED** [review]

File: runbooks/CI_pipelines/adding-ci-dockerhub-users.md [hunk 1]
```diff
@@ -74,11 +74,97 @@ there are 3 main steps:
...
+    ```
+
+2. Use the aws_secretsmanager_secret_version data source to provide the secret string as an environment variable to CircleCI. You can use the opg-org-infra repository module to create environment variables for CircleCI. Applying this module will set a value of `default` for each secret, to be replaced later.
+
+    ```hcl
+    data "aws_secretsmanager_secret_version" "product_dockerhub_id" {
+      secret_id = module.product.aws_secretsmanager_secret_dockerhub_id.id
+      provider  = aws.management
+    }
+
```
Comment (jamesrwarren) [inline runbooks/CI_pipelines/adding-ci-dockerhub-users.md:102]: Can you just add in the token here please as well :)

File: runbooks/CI_pipelines/adding-ci-dockerhub-users.md [hunk 2]
```diff
@@ -74,11 +74,97 @@ there are 3 main steps:
...
-5. You should now be able to reference these inside of the `config.yaml` for CircleCI.
-6. Follow the advice given in [using Docker Authenticated Pulls](https://circleci.com/docs/2.0/private-images/) to set up you pipeline with these credentials, when using environment variables.
+### 3. Add dockerhub credentials to a CircleCI Project
+
+1. Create AWS Secrets Manager secrets for the DockerID and access token in your product's module in opg-org-infra. Our intention is to create these secrets in the opg-management AWS account.
+
+    ```hcl
+    module "ci_dockerhub_secrets" {
+      source         = "../modules/ci_dockerhub_secrets"
+      product_prefix = local.product_prefix
```
Comment (williamfalconeruk) [inline runbooks/CI_pipelines/adding-ci-dockerhub-users.md:84]: might want to clarify product prefix, and where this should go if not already present.

File: runbooks/CI_pipelines/adding-ci-dockerhub-users.md (hunk 1 above)
Comment (andrewpearce-digital) [reply runbooks/CI_pipelines/adding-ci-dockerhub-users.md:102]: done

File: runbooks/CI_pipelines/adding-ci-dockerhub-users.md (hunk 2 above)
Comment (andrewpearce-digital) [reply runbooks/CI_pipelines/adding-ci-dockerhub-users.md]: done

**Review by jamesrwarren — APPROVED** [review]

**Review by charlesmmarshall — APPROVED** [review]

**Pull request merged**